test/websocket/stream.cpp

```diff
@@ -911,21 +911,23 @@ class stream_test
                 ws.set_option(pong_callback{});
 
                 // send auto fragmented message
-                ws.set_option(auto_fragment_size(3));
-                ws.write(sbuf("Hello"));
+                ws.set_option(auto_fragment{true});
+                ws.set_option(write_buffer_size{8});
+                ws.write(sbuf("Now is the time for all good men"));
```

Comment:
This is still a little bit in flux, I'm going to wait until I have finished the permessage-deflate extension and then revise all the tests, and try to get coverage up.